### DISCUSSION TITLE
Game resolution per game

### DISCUSSION THREAD

**Discussion by mlostekk** [ChristopherKlay/StadiaEnhanced#25] [Ideas & Requests]
It would be great if the tool could remember the resolutions per game. My setup is for example a 1440p monitor where i play the following games:

RDR: 1080p / 60fps (above resolutions will lead to 30fps)
AC:O: 1440p / 30fps (any resolution only brings 30fps, so i go max res instead of max fps)

This leads to the situation that i have to switch everytime i start a different game.

**Comment by ChristopherKlay** [comment]
This should definitely be possible. Added to the to-do list. (:
